src/lib/mina_base/account_timing.ml

```diff
@@ -83,10 +83,10 @@ end
 
 type as_record =
   ( bool
-  , Global_slot_since_genesis.Stable.V1.t
-  , Global_slot_span.Stable.V1.t
-  , Balance.Stable.V1.t
-  , Amount.Stable.V1.t )
+  , Global_slot_since_genesis.Stable.Latest.t
```

Comment:
Can these types omit the `Stable.Latest`, so `Global_slot_since_genesis.t`, for example. That's the same type.